### PR TITLE
find prime numbers

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: curriculum/js1/7.js [hunk 1]
```diff
@@ -5,8 +5,12 @@
...
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (num, i=2) => {
+  if (i === num) { return true}
+  if (num%i === 0){
+    return false;
+  }
+  return solution (num, i+1);
```
Comment (songz) [inline curriculum/js1/7.js:16]: would your solution work if num is 1?

Comment (saumya665) [reply curriculum/js1/7.js:16]: fixed it!

File: curriculum/js1/7.js [hunk 2]
```diff
@@ -5,10 +5,18 @@
...
+    return false;
+  }
+  if (i === num) { return true}
+  if (num%i === 0){
+    return false;
+  }
+  return solution (num, i+1);
 };
 
+console.log(solution(1));
```
Comment (songz) [inline curriculum/js1/7.js]: remove console.log

**Review by roycep — APPROVED** [review]
Just clean up line 12

**Pull request merged**